Resources/Prototypes/Nuclear14/Entities/Objects/Weapons/Melee/longblades.yml

```diff
@@ -10,3 +10,75 @@
   - type: Item
     size: 15
     sprite: Nuclear14/Objects/Weapons/Melee/machete.rsi
+
+- type: entity
+  name: ripper
+  parent: BaseItem
+  id: N14ripper
+  description: A light one handed wield chainsaw, originally designed by the pre-war army sappers to clean selvatic and wooded paths, it also works well against meat.
+  components:
+  - type: Sharp
```

Comment:
same stuff again, parent these either to a base or to one another to cut down on duplicated components if they're the same.